ESPixelStick/src/WiFiMgr.cpp

```diff
@@ -361,7 +362,8 @@ void c_WiFiMgr::AnnounceState ()
 
     String StateName;
     pCurrentFsmState->GetStateName (StateName);
-    // logcon (String (F ("WiFi Entering State: ")) + StateName);
+    LOG_PORT.println ("");
+    logcon (String (F ("WiFi Entering State: ")) + StateName);
```

Comment:
Do we need to log state to the console? I had removed this before when cleaning up the serial output.